dashboard/app/dsl/script_dsl.rb

```diff
@@ -45,6 +46,10 @@ def trophies(trophies_string)
     @trophies = ActiveRecord::ConnectionAdapters::Column::value_to_boolean(trophies_string)
   end
 
+  def wrapup_video(wrapup_video)
```

Comment:
(optional, if you like to take advantage of opportunities to legitimately use method_missing ;))